javascript-timers/interval.js

```diff
@@ -1,14 +1,13 @@
 var $h1 = document.querySelector('h1');
 setInterval(countdown, 1000);
+var timer = 4;
 
 function countdown(second) {
-  if ($h1.textContent === '4') {
-    $h1.textContent = '3';
-  } else if ($h1.textContent === '3') {
-    $h1.textContent = '2';
-  } else if ($h1.textContent === '2') {
-    $h1.textContent = '1';
-  } else if ($h1.textContent === '1') {
+  timer -= 1;
+  if ($h1.textContent === '1') {
     $h1.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(1);
```

Comment:
Instead of passing `1` directly in, you should save the id that is returned from calling `setInterval`